app/message-attachments/client/messageAttachment.html

```diff
@@ -1,5 +1,5 @@
 <template name="messageAttachment">
-	<div class="attachment">
+	<div class="attachment {{# if temp}}attachment-temp{{/if}}">
```

Comment:
Please provide a screenshot of this UI change.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/17441#issuecomment-671018953
The initial blur to indicate that the attachment is not delivered yet.